src/components/_profile.scss

```diff
@@ -0,0 +1,119 @@
+@use '../base.scss';
+
+.profile--dark {
+  background-color: base.$secondary-color;
+  color: base.$font-white;
+  line-height: 1.7em;
+  padding-bottom: 0.8rem;
+  padding-top: 0.8rem;
+}
```

Comment:
Please add these lines to stretch the profile:

`margin: 0 -15px;`
`padding-left: 15px;`
`padding-right: 15px;`